class/UserAccessManager.php

```diff
@@ -155,6 +155,9 @@ public function getFromCache($sKey)
     public function getUser($sId)
     {
         if (!isset($this->_aUsers[$sId])) {
+            if (!function_exists('get_userdata')) {
```

Comment:
I just wanted to have it in a single place, since it was checked twice in that file. But I would agree, if this is what you wanted, to have it in the main plugin file, where all of that logic is handled. I will modify the pull request.
https://github.com/GM-Alex/user-access-manager/blob/master/class/UserAccessManager.php#L568
https://github.com/GM-Alex/user-access-manager/blob/master/class/UserAccessManager.php#L738